src/Host/Client/Impl/Definitions/IRSessionProvider.cs

```diff
@@ -9,6 +9,7 @@
 namespace Microsoft.R.Host.Client {
     public interface IRSessionProvider : IDisposable {
         Uri BrokerUri { get; }
+        string BrokerName { get; }
```

Comment:
Ok. Or maybe IBrokerClient, cause it isn't a broker itself

---
In reply to: [78041694](https://github.com/Microsoft/RTVS/pull/2268#discussion_r78041694) [](ancestors = 78041694)